src/runtime/components/Table.vue

```diff
@@ -229,7 +238,7 @@ defineExpose({
       <tbody :class="ui.tbody({ class: [props.ui?.tbody] })">
         <template v-if="tableApi.getRowModel().rows?.length">
           <template v-for="row in tableApi.getRowModel().rows" :key="row.id">
-            <tr :data-selected="row.getIsSelected()" :data-expanded="row.getIsExpanded()" :class="ui.tr({ class: [props.ui?.tr] })">
+            <tr :data-selected="row.getIsSelected()" :data-can-select="!!props.onSelect" :data-expanded="row.getIsExpanded()" :class="ui.tr({ class: [props.ui?.tr] })" @click="handleRowSelect(row, $event)">
```

Comment:
nice! but out of curiosity, how would this work for accessibility / screenreaders as this behaves like a button?

Reply:
Maybe this could be solved if we add this attribute?

:role="props.onSelect ? 'button' : undefined"

Reply:
It might need `tabIndex="0"` too